src/common/datetime/format_date_time_with_seconds.ts

```diff
@@ -0,0 +1,23 @@
+import fecha from "fecha";
+
+// Check for support of native locale string options
+function toLocaleStringSupportsOptions() {
+  try {
+    new Date().toLocaleString("i");
+  } catch (e) {
+    return e.name === "RangeError";
+  }
+  return false;
+}
+
+export default toLocaleStringSupportsOptions()
+  ? (dateObj: Date, locales: string) =>
+      dateObj.toLocaleString(locales, {
+        year: "numeric",
+        month: "long",
+        day: "numeric",
+        hour: "numeric",
+        minute: "2-digit",
+        second: "2-digit",
+      })
+  : (dateObj: Date) => fecha.format(dateObj, "haDateTime");
```

Comment:
This doesn't work ? If locale string "i" is not supported, we fall back to `haDateTime` which does not have seconds.

Reply:
Good catch. But problem here is even worse. `haDateTime` was deleted with `util.js` back in 2018 in #1183. And after almost 2 years no one even noticed :)

Reply:
awkward 😅